### PR TITLE
Docs/getting started

### PR DESCRIPTION
**Pull request by frontend-specialisten**
## References

- #0000

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Prerequisites
+
+- [We recommend using VOLTA to manage your JavaScript command-line tools](volta.sh)
+- Node.js® version 14.18.1
+- Yarn - Please use Yarn v1 instead of npm to install dependencies  
+  `npm install --global yarn@1.22.17`
```
Comment (dzhavat) [inline docs/getting-started.md:6]: suggestion: In order to minimize setup differences between machines, I think we should promote Volta more and say that's how work on UI should be done. So my suggestion is to remove the lines about Node.js and Yarn and ask people to install Volta. My understanding is that Volta will read the `package.json` file and figure out on its own which versions of Node.js and Yarn to install.

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+## Visual Studio Code - recommended extensions
+
+- Angular Language Service
+- Prettier - Code formatter
+- Nx Console
+- EditorConfig for VS Code
+- markdownlint
+- ESLint
```
Comment (dzhavat) [inline docs/getting-started.md:15]: suggestion: Can we add these extensions to the `extensions.json` file under `.vscode` folder in the repo so VS Code prompts the developer to install them?

File: .gitignore [hunk 3]
```diff
@@ -28,7 +28,6 @@
 *.userosscache
 *.sln.docstates
 *.idea*
-.vscode/
```
Comment (frontend-specialisten) [inline .gitignore:31]: FYI: We exclude other files in .vscode later in this file

File: docs/getting-started.md (hunk 2 above)
Comment (frontend-specialisten) [reply docs/getting-started.md:12]: They are added now, should we just remove this section?

Comment (dzhavat) [reply docs/getting-started.md:12]: Yes, I think so. Can be replaced with a sentence saying that the extensions exist and how to install them (I think there's a command in VS Code that can install the recommended extensions if the file exists).

**Review by dzhavat — APPROVED** [review]

**Pull request merged**